lib/stm32wba/BLE_TransparentMode/Core/Inc/app_conf.h

```diff
@@ -130,7 +65,7 @@
 #define CFG_BLE_OPTIONS             (0 | \
                                      0 | \
                                      0 | \
-              BLE_OPTIONS_EXTENDED_ADV | \
+                                     0 | \
```

Comment:
BLE_OPTIONS_EXTENDED_ADV should be added to have same BLE features enabled

Reply:
ok, I will add it, thanks!